it/it-tests/src/test/java/util/ItUtils.java

```diff
@@ -338,9 +341,35 @@ public static ComponentNavigation getComponentNavigation(Orchestrator orchestrat
     return ComponentNavigation.parse(content);
   }
 
-  public static File findFileInClasspath(String classpathRelativeFilename) throws URISyntaxException {
-    ResourceLocation location = FileLocation.ofClasspath("/organization/IssueAssignTest/one-issue-per-file-profile.xml");
-    return new File(ItUtils.class.getResource(location.getPath()).toURI());
+  public static void restoreProfile(Orchestrator orchestrator, String classpathRelativeFilename) {
+    restoreProfile(orchestrator, classpathRelativeFilename, null);
+  }
+
+  public static void restoreProfile(Orchestrator orchestrator, String classpathRelativeFilename, String organization) {
+    WsClient adminWsClient = newAdminWsClient(orchestrator);
+    restoreProfile(adminWsClient, classpathRelativeFilename, organization);
+  }
+
+  public static void restoreProfile(WsClient adminWsClient, String classpathRelativeFilename, String organization) {
+    adminWsClient
+      .qualityProfiles()
+      .restoreProfile(
+        RestoreWsRequest.builder()
+          .setBackup(findFileInClasspath(classpathRelativeFilename))
+          .setOrganization(organization)
+          .build());
+  }
+
+  public static File findFileInClasspath(String classpathRelativeFilename) {
```

Comment:
Using `FileLocation` does not bring any value here. Using `Class#getResource()` is good enough for tests.